seed/views/v3/portfolio_manager.py

```diff
@@ -405,21 +412,23 @@ def generate_and_download_template_report(self, matched_template):
             response.headers))
 
         # Now we need to wait while the report is being generated
-        url = 'https://portfoliomanager.energystar.gov/pm/reports/reportData'
         attempt_count = 0
         report_generation_complete = False
         while attempt_count < 10:
             attempt_count += 1
 
             # get the report data
             try:
-                response = requests.get(url, headers=self.authenticated_headers)
+                json_authenticated_headers = self.authenticated_headers.copy()
+                json_authenticated_headers['Accept'] = 'application/json'
+                json_authenticated_headers['Content-Type'] = 'application/json'
+                response = requests.get(self.REPORT_URL, headers=json_authenticated_headers)
             except requests.exceptions.SSLError:
                 raise PMExcept('SSL Error in Portfolio Manager Query; check VPN/Network/Proxy.')
             if not response.status_code == status.HTTP_200_OK:
                 raise PMExcept('Unsuccessful response from report template rows query; aborting.')
 
-            template_objects = self.parse_template_response(response.text)['customReportsData']
+            template_objects = response.json()['reportTabData']
```

Comment:
we are being presumptuous that this key exists, but it has worked in the past, so leaving.